python/dev.py

```diff
@@ -255,6 +260,12 @@ def main() -> None:
         action="store_true",
         help="Install Python dependencies without cache",
     )
+    build_parser.add_argument(
+        "--glide-version",
+        type=str,
+        default="unknown",
+        help="Specify the sync client version (meant to be used in CD)",
```

Comment:
(rewrite it with cgpt): "Build the client with the provided version to define how the client identifies it's client version to the server and appear in the CLIENT INFO output"